README.md

```diff
@@ -42,6 +43,10 @@ before running them.
 Version History
 ---------------
 
+0.2.11 (06/01/2020)
+
+Added method to get a user by an access token - bjoernHeneka
+
```

Comment:
Can you format this like the other lines (indented and link your username)

Reply:
Also this has bjoernHeneka's username, please change that to your own.

Reply:
@pmill Could you please give me access to push my branch? My this merge request can be ignored

Reply:
You just need to push to your own branch on your fork and the PR will be updated.

Reply:
Thanks & done :)